console/console.go

```diff
@@ -260,6 +260,7 @@ func (c *Console) initPersonal(vm *goja.Runtime, bridge *bridge) {
 	if personal == nil || c.prompter == nil {
 		return
 	}
+	log.Error("Enabling deprecated personal namespace")
```

Comment:
This should be a warning, errors are for internal errors where things went very wrong.